mkdocs.yml

```
@@ -0,0 +1,87 @@
+# Configuration for all pages
+site_name: "MCAP"
+theme:
+  name: material
+  favicon: img/favicon32.png
+  logo: img/mcap.png
+  palette:
+    - scheme: default
+      toggle:
+        icon: material/brightness-7
+        name: Switch to dark mode
+      primary: "white"
+    - scheme: slate
+      toggle:
+        icon: material/brightness-4
+        name: Switch to light mode
+      primary: "black"
```

Docs suggest that with the `media` settings, we can automatically adjust to the user's selected scheme when they first visit the site: https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#system-preference
```suggestion
    - media: "(prefers-color-scheme: light)"
      scheme: default
      toggle:
        icon: material/brightness-7
        name: Switch to dark mode
      primary: "white"
    - media: "(prefers-color-scheme: dark)"
      scheme: slate
      toggle:
        icon: material/brightness-4
        name: Switch to light mode
      primary: "black"
```